ocs_ci/helpers/helpers.py

```
@@ -67,7 +68,8 @@
     get_ocs_build_number,
 )
 from ocs_ci.utility.utils import convert_device_size
-
+from ocs_ci.utility.templating import dump_data_to_temp_yaml
+from ocs_ci.helpers import helpers
```

Ruff (0.16.1)

[warning] Module `ocs_ci.helpers.helpers` imports itself

(PLW0406)